### ISSUE TITLE
Add "Open terminal" button to each database on connection

### ISSUE THREAD

**Issue by MurzNN** [cweijan/vscode-database-client#247]
In PostgreSQL cli when the database is not set, it tries connect to database, matching to passed username, and fails on this:
```
set "PGPASSWORD=superPassW0RD" && psql -U murz -h 127.0.0.1 -p 5432 
Password for user murz: 
psql: FATAL:  database "murz" does not exist
```
As result, we must manually reconnect with filling right database name.
To solve this problem will be good to add "Open terminal" button to each database line in list of connection databases.
Ad for main terminal button (without selecting exact database) - maybe add `postgres` database name?

**Comment by cweijan** [comment]
Fix at version 3.9.5.

**Comment by MurzNN** [comment]
Works well, thanks for quick implementing!
